enclave/core/sgx/backtrace.c

```diff
@@ -178,6 +229,11 @@ char** oe_backtrace_symbols_impl(
     if (argv)
         free_fcn(argv);
 
+    if ((oe_result_t)retval == OE_UNSUPPORTED)
+        OE_TRACE_WARNING(
+            "In-enclave backtrace is not supported. To enable, please import "
+            "debug.edl.\n");
```

Comment:
Shouldn't it say something like import "*" from debug.edl?   The developer can't just import the file, they have to specify which APIs to import from it.

Reply:
My intention is informing the user to take the action rather than referring to the exact line of code they need to add. Do you think the latter is more appropriate?

By the way, oeedger8r actually supports the syntax of `import debug.edl`, which does the same as `from debug.edl import *`.  Refer to @anakrish's great [document](https://github.com/openenclave/oeedger8r-cpp/blob/master/docs/Grammar.md).

Reply:
Updated the message to be more specific

Reply:
Interesting, I didn't know you could just import debug.edl.  All the samples do the more verbose version, so I'm not sure how developers would discover the shorter version, as it's not referenced in any OE getting started doc that I can find (the doc you mention is in a different project).

Reply:
So yes, I think the latter is more appropriate since as is, it is not clear to the developer what to do.

Reply:
Updated. Now the example of the log is
```
ssh_office@oe-mingwei-1804-8c:~/workspace/oe-ms2/build/tests/backtrace$ OE_LOG_LEVEL=WARNING ./host/backtrace_host enc/backtrace_enc 
=== test()
2020-07-01T18:19:25.000000Z [(E)WARN] tid(0x7f3dc9767740) | backtrace_enc:In-enclave backtrace is not supported. To enable, please add 

from "openenclave/edl/sgx/debug.edl" import *;

in the edl file.
```